pkg/virtualization/core/service/virtualmachine.go

```diff
@@ -272,3 +272,31 @@ func (vmms *VirtualSystemManagementService) SetMemoryMB(vm *virtualsystem.Virtua
 	err = vmms.ModifyVirtualSystemResourceEx(mem.WmiInstance, -1)
 	return
 }
+
+// Re-implementation of Disable-VMConsoleSupport Cmdlet
+// Removes the Synthetic Mouse, Synthetic Keyboard, and Synthetic Display devices from a VM
+func (vmms *VirtualSystemManagementService) RemoveHIDDevices(vm *virtualsystem.VirtualMachine) (err error) {
+	// ResourceSubType consts. Note: Cannot query based on ResourceType because Mouse and Keyboard share ResourceType: 13
+	const SYNTHETIC_MOUSE_SUBTYPE = "Microsoft:Hyper-V:Synthetic Mouse"
```

Comment:
can you move these to 
https://github.com/microsoft/wmi/blob/93a9772576492c54893cd6586faa72cdf49a2937/pkg/virtualization/constant/constant.go